c2corg_api/models/outing.py

```diff
@@ -78,14 +78,46 @@ class _OutingMixin(object):
 
     disable_comments = Column(Boolean)
 
+    hiking_rating = Column(enums.hiking_rating)
+
+    ski_rating = Column(enums.ski_rating)
+
+    labande_ski_rating = Column(enums.labande_ski_rating)
```

Comment:
According to https://github.com/c2corg/v6_api/issues/685#issuecomment-336477904 there was a mistake in the specs: we need ``labande_global_rating`` instead of ``labande_ski_rating``.

Reply:
updated